### ISSUE TITLE
cf api fails after reboot

### ISSUE THREAD

**Issue by frankr6591** [cloudfoundry/cli#701]
I am on Windows + Cygwin using cf.exe version 6.14.0+2654a47-2015-11-18
I had done cf api 2 days ago on my laptop and everything was working.   But I rebooted windows and  now it is timeouts... need help.

$ cf api https://api.ng.bluemix.net
Setting api endpoint to https://api.ng.bluemix.net...

REQUEST: [2015-12-02T07:39:49-06:00]
GET /v2/info HTTP/1.1
Host: api.ng.bluemix.net
Accept: application/json
Authorization: [PRIVATE DATA HIDDEN]
Content-Type: application/json
User-Agent: go-cli 6.14.0+2654a47 / windows

FAILED
Error performing request: Get https://api.ng.bluemix.net/v2/info: dial tcp 75.126.70.44:443: i/o timeout
FAILED
Error performing request: Get https://api.ng.bluemix.net/v2/info: dial tcp 75.126.70.44:443: i/o timeout

**Comment by krishicks** [comment]
Hi @frankr6591,

Cygwin isn't currently a supported shell for using the CF CLI. Please try again with Powershell or Command Prompt and let me know how it goes.

**Comment by frankr6591** [comment]
Kris,  cygwin is not important.    I am using the CF Window (64 bit)… the problem occurs on Window (DOS cmd line). 

 

I am restarting the system to investigate if there is something in my system… 

 

NOTE: the system locked up (on wake-up) and so I had to reboot with hard. 

 

Regards,

Frank Rojas

512-422-0210

**Comment by frankr6591** [comment]
Kris, thanks... I restarted the system and it still failed.
Ran from both DOS cmd window and cygwin terminal. 

I re-installed  CF (cf_installer.exe) and it still failed but now I get different message & trace was turned off:

$ cf login -a https://api.ng.bluemix.net
API endpoint: https://api.ng.bluemix.net
FAILED
ConnectEx tcp: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.

**Comment by krishicks** [comment]
@frankr6591 Are you using a proxy?

**Comment by frankr6591** [comment]
No, yesterday the cf cli was working fine…  w/o proxy to my knowledge… 

 

Anyway to debug?

 

Thanks,

Frank

**Comment by krishicks** [comment]
@frankr6591 The CLI stores a configuration file in `$HOME/.cf/config.json`. You might try deleting that file to remove any bad data that might have been stored there.

**Comment by frankr6591** [comment]
Kris, thanks... I have a record open with bluemix but still nothing seems to work.  FYI, I have cf installed on W: drive and $HOME was set to W:/myhome... but the .cf was created in C:/Users/IBM_ADMIN/.cf. I deleted config.json and plugins and still it fails... 

FYI, we've traced via cf config -trace true... and still timeout.   I believe the request is making it into bluemix. Ping works and accesses the server. 

I believe we must be missing something very obvious??? Thanks for the suggestion...
